build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,136 @@
+# Coreth ANTs and ARC-20s
+
+## ANTs on the C-Chain
+
+### Why move an ANT from the X-Chain to the C-Chain?
+
+An Avalanche Native Token (ANT) is any token created on the X-Chain. These tokens can be exchanged on the X-Chain at lightning fast speeds taking advantage of the superior performance offered by using a DAG instead of a strictly linear chain.
+
+When we want to use smart contract functionality, we need a total ordering of state transitions (transactions), so that each transaction can modify the state of the network without having to worry about external dependencies.
+
+Therefore, when we want to use smart contract functionality, we must perform an atomic swap to move ANTs from the X-Chain to the C-Chain.
+
+### ANTs on the C-Chain
+
+#### Native Coin - AVAX
+
+ANTs on the C-Chain are either AVAX &emdash; the Avalanche native asset &emdash; or another asset that was created on the X-Chain.
```

Comment:
Looks like the emdash isn't showing up the way it's intended

Reply:
Thanks! Just updated